etc/admin/models.py

```diff
@@ -19,6 +19,8 @@ class MyPage(CustomModelPage):
 
             title = 'Test page 1'  # set page title
 
+            bound_admin = MyAdmin  # set admin class
```

Comment:
Bound admin is ment for an instance of a class.
We could add ``admin_cls`` with default of `CustomPageModelAdmin` to put classes there.